deploy/html/viur_base.html

```diff
@@ -117,14 +117,13 @@
 				{% block content %}
 					{# put your content here #}
 				{% endblock content %}
-			</article><!-- content -->
-
-			<footer>
-				{% block footer %}
-					{# put your footer here #}
-				{% endblock footer %}
-			</footer>
 
+				<footer>
+					{% block footer %}
+						{# put your footer here #}
+					{% endblock footer %}
+				</footer>
+			</div><!-- content -->
```

Comment:
Based on my comment above, it is more correct to place the footer outside the main content again. Semantically, the footer refers to the entire page and not just the main area. The CSS would then have to be adapted accordingly.

Reply:
Since the this applies imo also to the header, I have moved both elements out of the main content (and out of the horizontal flex box which causes this behavior.)

Now footer and header are using the hole width of the visible area. I am going to take care of this too :)

Reply:
Ok, to solve this Problem, I decided to leave footer and header in the div assigned to the content class and wrapped an additional main element around the jinja content placeholder.

To make it clear that the div is only a functional block, we could rename the content class. What do you think?

Reply:
The solution works fine. We can use it as is for now, without renaming `content` imho. 
We should definitely look into the workings of `wrap`, `bind`, `content` and `section` — because they should go hand in hand and provide an easy to use framework especially for small websites. Perhaps it is even enough to reduce `bind flex` to `bind`.